src/test/java/io/vertx/junit5/VertxTestContextTest.java

```diff
@@ -456,4 +457,15 @@ void check_unsatisifed_checkpoint_callsites() {
     b.flag();
     assertThat(context.unsatisfiedCheckpointCallSites()).isEmpty();
   }
+
+  @Test
+  @DisplayName("Check failNow() called with a string")
+  void check_fail_now_called_with_a_string() {
+    VertxTestContext context = new VertxTestContext();
+
+    context.failNow("error message");
+
+    assertThat(context.failed()).isTrue();
+    assertThat(context.causeOfFailure()).isInstanceOf(NoStackTraceThrowable.class);
```

Comment:
You should also chain `hasMessage` to verify the exception message 😉

Reply:
right.
done :)